performance/src/efficiency/finding_performance_writer.cpp

```diff
@@ -170,14 +170,15 @@ void finding_performance_writer::write_common(
         std::vector<particle_hit_count> particle_hit_counts =
             identify_contributing_particles(measurements, evt_map.meas_ptc_map);
 
-        if (particle_hit_counts.size() == 1) {
-            auto pid = particle_hit_counts.at(0).ptc.particle_id;
+        // Consider it being matched if hit counts is larger than the half
+        // of the number of measurements
+        assert(measurements.size() > 0u);
+        if (particle_hit_counts.at(0).hit_counts > measurements.size() / 2) {
```

Comment:
Can we make this configurable?

Usually we define:
- loose (0.5)
- thight (0.7)
- full (1.0)
matching criteria, it would be good to be able to reproduce those.

Reply:
Yup. Added `matching_ratio` now